blueprints/ember-frost-object-browser/index.js

```diff
@@ -11,8 +11,8 @@ module.exports = {
   afterInstall: function () {
     return this.addAddonsToProject({
       packages: [
-        {name: 'ember-frost-core', target: '>=0.0.14 <2.0.0'},
-        {name: 'ember-frost-bunsen', target: '^3.0.0'},
+        {name: 'ember-frost-core', target: '>=0.1.0 <2.0.0'},
```

Comment:
Do we actually rely on any changes to `ember-frost-core` since version `0.0.14`? If not I don't know that this change makes sense.